### PR TITLE
add class and methods to transform world map to Gall Peters projection

### PR DESCRIPTION
**Pull request by kvelon**
change target of pull request to 'world' branch

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,7 @@
 *.pyc
-**/__pycache__
\ No newline at end of file
+**/__pycache__
+internal/web.py.bak
+.idea/
+internal/static/cartdata/
```
Comment (jansky) [inline .gitignore:5]: Please remove this from the .gitignore. These files must be tracked as a part of the repository.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,7 @@
...
+data/
```
Comment (jansky) [inline .gitignore:6]: Please remove this from the .gitignore. These files must be tracked as a part of the repository.

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,7 @@
...
+internal/handlers/
```
Comment (jansky) [inline .gitignore:7]: Please remove this from the .gitignore. These files must be tracked as a part of the repository.

File: internal/web.py [hunk 4]
```diff
@@ -160,7 +162,12 @@
 'andorra': andorra.CartogramHandler(),
 'ethiopia': ethiopia.CartogramHandler(),
 'myanmar': myanmar.CartogramHandler(),
+<<<<<<< HEAD
```
Comment (jansky) [inline internal/web.py:165]: Please remove this line, as well as 168 and 170 to complete the merge.

**Comment by kvelon** [comment]
Hello Ian,

I thought it would have been better if I didn't include the unfinished world map and its associated files and lines of code and so I added those items to .gitignore. But I suppose that since I am making this pull request to a branch, it is fine for those files to be kept, and that it would allow you to clone the branch and view the world map without further hassle!

I have since removed those items from .gitignore and pushed the changes.

**Pull request merged**